src/includes/configuration/config-intro/unity.mdx

```diff
@@ -1,23 +1,20 @@
-<Alert level="warning" title="Note">
+In order to provide native crash support, the Sentry SDK for Unity includes platform-specific (that is, Native) SDKs, such as [Android](/platforms/android/), [Apple](/platforms/apple/guides/ios/), and [Native](/platforms/native/).
 
-The native support is currently limited to the configuration via the editor window. This is due to the SDK relying on the configuration options to be available at build time to set up the Android and iOS SDKs.
-  
-</Alert>
+By default, we initialize the native SDKs before Unity itself runs and the Sentry SDK with the [BeforeSceneLoad RuntimeInitializeOnLoadMethodAttribute](https://docs.unity3d.com/ScriptReference/RuntimeInitializeLoadType.BeforeSceneLoad.html). This allows us to capture bugs/crashes of the engine itself, any native plugin, and anything written in C#. The setup for the native SDKs happens during build time, which is why we rely on the options being set in the Sentry editor configuration window and saved to `Assets/Resources/Sentry/SentryOptions.asset`. To provide a way to modify options programmatically, we've added `ScriptableOptionsConfiguration` to the `Option Config`.
```

Comment:
```suggestion
By default, we initialize the native SDKs before the Unity engine itself. That means it also runs before the C# layer of the Sentry SDK, because it relies on [BeforeSceneLoad RuntimeInitializeOnLoadMethodAttribute](https://docs.unity3d.com/ScriptReference/RuntimeInitializeLoadType.BeforeSceneLoad.html) to get initialized. This allows us to capture bugs/crashes of the engine itself, any native plugin, and anything written in C#. The setup for the native SDKs happens during build time, which is why we rely on the options being set in the Sentry editor configuration window and saved to `Assets/Resources/Sentry/SentryOptions.asset`. To provide a way to modify options programmatically, we've added `ScriptableOptionsConfiguration` to the `Option Config` tab in the Sentry editor window. 

You can click the button _Create options configuration_ to get the C# scriptable object added to your project.
```